ee/ucp/admin/configure/deploy-route-reflectors.md

```diff
@@ -224,7 +219,7 @@ You can learn more about this configuration in the
 ## Stop calico-node pods
 
 If you have `calico-node` pods running on the nodes dedicated for running the
-Route Reflector, manually delete them. This ensures that you don't have them
+route reflector, manually delete them. This ensures that you don't have them
```

Comment:
Change the line to read:

"Manually delete any `calico-node` pods that are running on nodes dedicated to the running of route reflectors.